### ISSUE TITLE
[A11y] Homepage - Semantics - Emphasis

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1914]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

The following text is visually emphasized:

> A collaboration between the University of Oregon Libraries and Oregon State University Libraries & Press, Oregon Digital comprises more than 100,000 digitized works including historic and modern photographs, manuscripts, and publications.

It seems this is emphasized by the font, not CSS and not a semantic tag.  This makes it extra tricky, because if we add an `<em>` tag, the text will change to have a light blue background for some reason.  But if we don't add that tag, we're offering a visual emphasis where there is no semantic emphasis.

### Suggested Fix

Use a standard font and the `<em>` tag if possible.  Otherwise this probably needs to go to somebody with more expertise to figure out how important it is to make sure emphasis is properly tagged.  I've heard anecdotes that this isn't that important, but that doesn't mean it's not a hard failure - I'm just not sure.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

You have to verify that the markup is correct, not just the visual display, so this needs at least some familiarity to check that `<em>` is in use.

**Comment by CGillen** [comment]
I don't think `<i>` or `<em>` makes sense here.
We're definitely not using stress emphasis on the text. We're not insisting the statement is true in an argument, or that we urgently need to tell people about our collaboration. Usually someone would use one or two words in an `<em>` tag
`Cats are <em> cute </em> animals` (as apposed to `scary`) or `Cats are cute <em> animals </em>` (as apposed to `plants`) changes what the sentence means. For the full sentence you would use `<em>Cats are cute animals!</em>` to either suggest we're fighting with someone in a larger conversation, or if it's a danger to the user to not understand cats and their cuteness.

`<i>` is also loosely semantic, it's used for names, character thoughts, citations (with proper <cite> tags), foreign language (with lang attribute), technical terms, etc.

We're not doing anything that fit those cases, and the text reads no different with the italics to a non-sr user. The best argument would be that it draws a non-sr user's attention a little better but I don't think it's an urgent piece of information. Not using `<i>` or `<em>` is actually the best way to represent this text.

**Comment by CGillen** [comment]
https://blog.logrocket.com/youre-using-em-wrong/
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-em-element

**Comment by jechols** [comment]
Perfect explanation, thanks for the links - and sorry for the wasted time on this one. Clearly I am exactly the person being described in that first blog article.
